source/slang/slang-ast-support-types.h

```diff
@@ -128,6 +128,13 @@ namespace Slang
         kConversionCost_Impossible = 0xFFFFFFFF,
     };
 
+    typedef unsigned int BuiltinConversionName;
+    enum : BuiltinConversionName
+    {
```

Comment:
Nitpik: Why not just use enum class, then theres no need for the kSomeName_ prefixing?